### ISSUE TITLE
Tours: Build a way to pause a tour directly from the tooltip

### ISSUE THREAD

**Issue by theripper93** [foundryvtt/foundryvtt#7646]
### User Experience

Many Tours will require some kind of user interaction before proceding. While the tour can do that via code by subclassing, it would be a good user experience to have the user perform the actual operations.

Build a button in the tooltip to put the Tour on "hold". While a tour is on hold, a button should be displayed somewhere on the body (possibly in the location of the tooltip) to resume the tour, without the need to manually close the tour, perform the operation, reopen the Tour Manager and resuming the Tour

**Comment by aaclayton** [comment]
This feels a bit more like custom tour functionality to me than core tour functionality. Will leave this open for @cswendrowski thoughts.

**Comment by theripper93** [comment]
I feel like at the moment if you want to make a tour that does not simply involve showing around a form you will need to heavily customize the tour. This could scare away a lot of module devs from adding tours. This functionality I’m proposing would cover a very large use case for module tours

Eg:

please create an active effect then resume the tour

or 

create a drawing , open the configuration and resume the tour

and so on… the use cases are a lot

That said if you think the bare bone approach is the way to go I understand

**Comment by cswendrowski** [comment]
Re-summarizing conversations had in Discord -

The design intent is a series of increasing complexity. From Least to Most complex:
* Tooltips - Points to and gives clarifying info on a single UI element
* Tours - extends Tooltips, points to an ordered list of UI elements with clarifying info / descriptions
* Tutorials - would extend Tours, adding points of interaction and branching paths

V10 brought the first two, while Tutorials is something we feel we could better build after seeing the general usecases the community ends up extending Tours into themselves.

**Comment by theripper93** [comment]
Yep, all good. We'll keep this in mind for a bit and see where it goes
